packages/math-input/src/components/keypad-switch.tsx

```diff
@@ -11,13 +12,17 @@ type Props = {
     style?: StyleType;
 
     useV2Keypad?: boolean;
+    onAnalyticsEvent: AnalyticsEventHandlerFn;
 };
 
 function KeypadSwitch(props: Props) {
     const {useV2Keypad = false, ...rest} = props;
 
     const KeypadComponent = useV2Keypad ? MobileKeypad : LegacyKeypad;
 
+    // Note: Although we pass the "onAnalyticsEvent" to both keypad components,
+    // only the current one uses it. There's no point in instrumenting the
+    // legacy keypad given that it's on its way out the door.
```

Comment:
I changed my mind!  I've got a follow-up PR that'll implement the analytics for the legacy keypad and remove this comment!